react/features/lobby/components/AbstractKnockNotification.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+
+import { PureComponent } from 'react';
+
+export type Props = {
+
+    /**
+     * Info about the avatar that helps us render on (e.g. URL, or avatar ID).
+     */
+    avatarInfo: string,
```

Comment:
string?

Reply:
All we send is the email ... so I suppose that is the string.

Reply:
SSorry I was a bit scant on details. Looking at this one wonders what it could contain, and thee comment doesn't help clarify that either...